chapter_06/lib/models/plan.dart

```diff
@@ -1,18 +1,17 @@
 import '../repositories/repository.dart';
-import 'package:flutter/foundation.dart';
 import './task.dart';
 
 class Plan {
   final int id;
-  String name = '';
+  String name;
   List<Task> tasks = [];
 
-  Plan({@required this.id, this.name = ''});
+  Plan({required this.id, this.name = ''});
 
   Plan.fromModel(Model model)
       : id = model.id,
-        name = model?.data['name'],
-        tasks = model?.data['task']
+        name = model.data['name'] ?? '',
```

Comment:
I was getting `type 'Null' is not a subtype of type 'String'`, so I decided to solve this way.